pkg/kubectl/.import-restrictions

```diff
@@ -122,6 +122,7 @@
                 "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util",
                 "k8s.io/kubernetes/pkg/scheduler/api",
                 "k8s.io/kubernetes/pkg/scheduler/cache",
+                "k8s.io/kubernetes/pkg/scheduler/internal/cache",
```

Comment:
Why is this needed?

Reply:
I think it's due to a transitive dependency. Obviously that package cannot be imported by kubectl, but this line was needed to make the tooling happy.

/cc @lavalamp

Reply:
Yeah, import boss doesn't understand about internal packages.

Reply:
Sigh...

Reply:
Since we can't do better ...
/approve